workitem/workitem_repository_blackbox_test.go

```diff
@@ -24,6 +29,20 @@ func TestRunWorkTypeRepoBlackBoxTest(t *testing.T) {
 	suite.Run(t, &workItemRepoBlackBoxTest{DBTestSuite: gormsupport.NewDBTestSuite("../config.yaml")})
 }
 
+// SetupSuite overrides the DBTestSuite's function but calls it before doing anything else
+func (s *workItemRepoBlackBoxTest) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
+
+	// Make sure the database is populated with the correct types (e.g. bug etc.)
+	if _, c := os.LookupEnv(resource.Database); c != false {
+		if err := models.Transactional(s.DB, func(tx *gorm.DB) error {
+			return migration.PopulateCommonTypes(context.Background(), tx, workitem.NewWorkItemTypeRepository(tx))
+		}); err != nil {
+			panic(err.Error())
```

Comment:
Not really, since the tests would definitely fail if the 'migration' doesn't happen. :)